pkg/crc/api/adapter.go

```diff
@@ -1,16 +1,18 @@
 package api
 
-import "github.com/code-ready/crc/pkg/crc/machine"
+import (
+	"github.com/code-ready/crc/pkg/crc/machine"
+)
 
 type AdaptedClient interface {
-	Delete(deleteConfig machine.DeleteConfig) DeleteResult
+	Delete(deleteConfig machine.DeleteConfig) Result
 	GetConsoleURL(consoleConfig machine.ConsoleConfig) (machine.ConsoleResult, error)
 	Start(startConfig machine.StartConfig) (machine.StartResult, error)
 	Status(statusConfig machine.ClusterStatusConfig) (machine.ClusterStatusResult, error)
-	Stop(stopConfig machine.StopConfig) (machine.StopResult, error)
+	Stop(stopConfig machine.StopConfig) Result
 }
 
-type DeleteResult struct {
+type Result struct {
 	Name    string
```

Comment:
This could have been named `Result` from the start, rather than being first `DeleteResult` and then `Result`

Reply:
done